draft-gruessing-moq-requirements.md

```diff
@@ -201,14 +218,20 @@ place.
 |**Senders/Receivers**|  One to Many
 |**Bi-directional**| No
 
-Where media is received from a live broadcast or stream. This may comprise of
-multiple audio or video outputs with different codecs or bitrates. This may also
-include other types of media essence such as subtitles or timing signalling
-information (e.g. markers to indicate change of behaviour in client such as
-advertisement breaks). The use of "live rewind" where a window of media behind
-the live edge can be made available for clients to playback, either because the
-local player falls behind edge or because the viewer wishes to play back from a
-point in the past.
+Where media is received from a live broadcast or stream either as a broadcast
+with fixed duration or as ongoing 24/7 output. The number of receivers may vary
+depending on the type of content; breaking news events may see sharp, sudden
+spikes, whereas sporting and entertainment events may see a more gradual ramp
+up with a higher sustained peak with some changes based on match breaks or
+interludes.
+
+Such broadcasts may comprise of multiple audio or video outputs with different
+codecs or bitrates, and may also include other types of media essence such as
+subtitles or timing signalling information (e.g. markers to indicate change of
+behaviour in client such as advertisement breaks). The use of "live rewind"
+where a window of media behind the live edge can be made available for clients
```

Comment:
I don't know if saying "a **limited** window of media behind the live edge" is worth saying, but it might help us with requirements for rewinding (say) 30 **seconds**, versus rewinding (say) 30 **minutes**. I've already gotten wrapped up between what relays keep around for retransmission, versus what caches provide for users who are rewinding - these seem like three different levels of "keeping something around in case you need it". 

I'll just ask this, if my daughter joins an hour and a half late for a London theater production with Benedict Cumberbatch, do you think a content provider/CDN that is serving that event using MOQ/WARP, would still want to serve very late joiners who want to watch from the beginning using MOQ/WARP, or using HTTP? That's not a relevant question for your PR, but it's a question that might be worth asking. 

And my daughter would **absolutely** want to join an hour and a half late, instead of being forced to 

- wait for the event to end (where HTTP is a fine answer), or 
- (potentially rewinding 10 minutes), watching to the end, and then watching the first hour and 20 minutes (potentially rewatching the parts she watched live). 

Because Benedict Cumberbatch is one of the reasons she flies to London. The End. :star_struck: